pkg/hintrunner/zero/zerohint_blake.go

```diff
@@ -83,3 +83,77 @@ func createBlake2sAddUint256BigendHinter(resolver hintReferenceResolver) (hinter
 
 	return newBlake2sAddUint256BigendHint(low, high, data), nil
 }
+
+func newBlake2sAddUint256Hint(low, high, data hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "Blake2sAddUint256",
+		Op: func(vm *VM.VirtualMachine, _ *hinter.HintRunnerContext) error {
```

Comment:
This method is practically the same as `Blake2sAddUint256Bigend`, I think the main difference is in the shift you have to use, is that correct? I would suggest then to unify them in a single method, you can use a flag to know if you want the result in big endian or little endian and calculate the correct `shift`. Something similar happened with `Uint256Add` and `Uint256AddLow` hints, you can check that code to see how it was handled

Reply:
Yep different shift and different positions in memory. I've updated the code.